src/Compilers/Core/CodeAnalysisTest/DefaultAnalyzerAssemblyLoaderTests.cs

```diff
@@ -83,5 +118,194 @@ public void AssemblyLoading()
 
             Assert.Equal(expected, actual);
         }
+
+        [ConditionalFact(typeof(CoreClrOnly))]
+        public void AssemblyLoading_AssemblyLocationNotAdded()
+        {
+            var loader = new DefaultAnalyzerAssemblyLoader();
+            loader.AddDependencyLocation(_testFixture.Gamma.Path);
+            loader.AddDependencyLocation(_testFixture.Delta1.Path);
+            Assert.Throws<FileNotFoundException>(() => loader.LoadFromPath(_testFixture.Beta.Path));
+        }
+
+        [ConditionalFact(typeof(CoreClrOnly))]
+        public void AssemblyLoading_DependencyLocationNotAdded()
+        {
+            StringBuilder sb = new StringBuilder();
+            var loader = new DefaultAnalyzerAssemblyLoader();
+            loader.AddDependencyLocation(_testFixture.Gamma.Path);
```

Comment:
Think this test would be easier to read quickly if you added a comment outlining explicitly that we're missing `Gamma` dependencies here.

Reply:
Done